src/styles/Home.module.scss

```diff
@@ -1,5 +1,9 @@
 @use "src/styles/medias";
 
+.display_none {
+  display: none !important;
```

Comment:
Precisava do !important mesmo ? não tem alguma forma de não precisar forçar ele ?

Reply:
Pelo visto não precisa, acho que tinha alguma imagem que estava sobressaindo sobre a tela de loading mas parece que foi corrigido :thinking:, vou remover esse !important